modules/third_party_perception/proto/third_party_perception_component.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto2";
+
+package apollo.third_party_perception;
+
+message ThirdPartyPerceptionDevice {
+  optional string device_names = 1 [default = "ThirdPartyPerceptionSmartereye"];
```

Comment:
It is probably better to use Enum here since you candidates are "ThirdPartyPerceptionSmartereye" and "ThirdPartyPerceptionMobileye", which are mutually exclusive.